server/app/services/MessageKey.java

```diff
@@ -130,12 +130,11 @@ public enum MessageKey {
   ERROR_NOT_FOUND_DESCRIPTION("error.notFoundDescription"),
   ERROR_NOT_FOUND_DESCRIPTION_LINK("error.notFoundDescriptionLink"),
   DATE_VALIDATION_INVALID_DATE_FORMAT("validation.invalidDateFormat"),
-  DATE_VALIDATION_MISFORMATTED("validation.dateMisformatted"),
   DAY_LABEL("label.day"),
+  DIALOG_DELETE_CONFIRMATION("dialog.deleteConfirmation"),
   DOB_EXAMPLE("label.dobExample"),
   DOB_ERROR_LABEL("label.errorDOB"),
   DOB_LABEL("label.dob"),
-  DIALOG_DELETE_CONFIRMATION("dialog.deleteConfirmation"),
```

Comment:
question (blocking): did you mean to delete this as well?

Reply:
this one just got moved above in the interest of keeping this alphabetized

Reply:
Ohh I see that now.